shell/platform/tizen/BUILD.gn

```diff
@@ -5,6 +5,9 @@
 import("//flutter/shell/platform/common/cpp/client_wrapper/publish.gni")
 import("//flutter/shell/platform/tizen/config.gni")
 
+# define enable_profiling
+import("//build/config/profiler.gni")
+
```

Comment:
You forgot to remove these lines.

Reply:
Oh,  sorry for that 🤦‍♂️ I've removed them now